src/I18n/DateFormatTrait.php

```diff
@@ -350,6 +350,24 @@ public function jsonSerialize()
     }
 
     /**
+     * Get the difference formatter instance or overwrite the current one.
+     *
+     * @param \Cake\I18n\RelativeTimeFormatter|null $formatter The formatter instance when setting.
+     * @return \Cake\I18n\RelativeTimeFormatter The formatter instance.
+     */
+    public function diffFormatter($formatter = null)
```

Comment:
Shouldn't this function be static? It only deals with static properties internally